cmake/modules/FindFFTW3.cmake

```diff
@@ -0,0 +1,22 @@
+# - Try to find FFTW3
+# Once done, this will define
+#
+#  FFTW3_FOUND - system has FFTW3
+#  FFTW3_INCLUDE_DIRS - the FFTW3 include directories
+#  FFTW3_LIBRARIES - link these to use FFTW3
+
+# INCLUDE(UsePkgConfig)
+
+FIND_PATH(FFTW3_INCLUDE_DIR fftw3.h)
```

Comment:
Please use `FindPugixml.cmake` as an example.
This should be more like
```
# - Try to find FFTW3
# Once done, this will define
#
#  FFTW3_FOUND - system has FFTW3
#  FFTW3_INCLUDE_DIRS - the FFTW3 include directories
#  FFTW3_LIBRARIES - link these to use FFTW3

include(LibFindMacros)

libfind_pkg_detect(FFTW3 fftw3
  FIND_PATH fftw3.h
  FIND_LIBRARY fftw3f libfftw3f
)

# Set the include dir variables and the libraries and let libfind_process do the rest.
# NOTE: Singular variables for this library, plural for libraries this this lib depends on.
set(FFTW3_PROCESS_INCLUDES FFTW3_INCLUDE_DIR)
set(FFTW3_PROCESS_LIBS FFTW3_LIBRARY)
libfind_process(FFTW3)
```